app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="day_button" parent="android:style/Widget.CompoundButton.CheckBox">
+        <item name="android:gravity">center_vertical|center_horizontal</item>
+        <item name="android:textColor">@color/black</item>
+        <item name="android:button">@null</item>
+        <item name="android:height">40dp</item>
+        <item name="android:width">40dp</item>
+        <item name="android:text">Day</item>
+        <item name="android:background">@drawable/day_button</item>
+        <item name="android:elevation">10dp</item>
+        <item name="android:autoSizeMaxTextSize">30dp</item>
+        <item name="android:autoSizeMinTextSize">6dp</item>
+        <item name="android:autoSizeStepGranularity">1dp</item>
+        <item name="android:autoSizeTextType">uniform</item>
+        <item name="android:paddingLeft">8sp</item>
+        <item name="android:paddingRight">8sp</item>
+    </style>
+</resources>
```

Comment:
These are the default values when creating the new checkboxes. Setting the style of a button to day_button automatically applies these values.